package.json

```
@@ -9,9 +9,9 @@
     "tsc": "tsc",
     "clean": "rimraf dist",
     "build": "npm run clean && npm run tsc && npm run lint",
-    "start": "npm run build &&export NODE_ENV=production&&nodemon ./src/index.ts",
-    "dev": "npm run build &&export NODE_ENV=development&&nodemon ./src/index.ts",
-    "debug": "npm run build&&export NODE_ENV=development&&nodemon --inspect ./src/index.ts 1313"
+    "start": "npm run build && (export NODE_ENV=production || set NODE_ENV=production) && nodemon ./src/index.ts",
+    "dev": "npm run build && (export NODE_ENV=production || set NODE_ENV=production) && nodemon ./src/index.ts",
+    "debug": "npm run build && (export NODE_ENV=production || set NODE_ENV=production) && nodemon --inspect ./src/index.ts 1313"
```

Good spot 🙌 Causes some issues on Linux though, since the conditional statement seems not to be executed accurately.

 ```suggestion
    "start": "npm run build && export NODE_ENV=production&& npm run nodemon-run",
    "dev": "npm run build && export NODE_ENV=production&& npm run nodemon-run",
    "debug": "npm run build && export NODE_ENV=production&& npm run nodemon-debug",
    "start-win": "npm run build && set NODE_ENV=production&& npm run nodemon-run",
    "dev-win": "npm run build && set NODE_ENV=production&& npm run nodemon-run",
    "debug-win": "npm run build && set NODE_ENV=production&& npm run nodemon-debug",
    "nodemon-run": "nodemon ./src/index.ts",
    "nodemon-debug": "nodemon --inspect ./src/index.ts 1313",
```